src/modules/seventv/utils.js

```diff
@@ -1,11 +1,11 @@
 import {hasFlag} from '../../utils/flags.js';
 import Emote from '../emotes/emote.js';
 
-function emoteUrl(id, version, static_ = false) {
-  return `https://cdn.7tv.app/emote/${encodeURIComponent(id)}/${version}${static_ ? '_static' : ''}.webp`;
+function emoteUrl(url, version, static_ = false) {
+  return `https:${url}/${version}${static_ ? '_static' : ''}.webp`;
 }
 
-export function createEmote(id, code, animated, owner, category, overlay) {
+export function createEmote(id, code, url, animated, owner, category, overlay) {
```

Comment:
rather than adding an arg to the middle of the list, we prefer to add new args onto the end of the list